common-lib/src/main/java/me/bristermitten/pdmlibs/http/HTTPCache.java

```diff
@@ -25,7 +25,7 @@ public HTTPCache(String userAgent)
 
 
     @NotNull
-    public byte[] fetch(@NotNull final URL url)
+    public byte @NotNull [] fetch(@NotNull final URL url)
```

Comment:
Is this intentional? Looks very odd...

Reply:
Odd yes, but quite intentional.

Reply:
Any benefit to this over `@NotNull byte[]`? If not, probably a good idea to change it - it looks pretty messy

Reply:
I ASSUME this also ensures the individual values in the array are also not null, however now that I'm thinking about it, byte is a primitive, it can't be null can it?

Reply:
Yeah, I assumed the same. I guess strictly it should be `@NotNull byte @NotNull []`, but since bytes can't be null it's unnecessary